axelar-chains-config/info/devnet-amplifier.json

```diff
@@ -778,6 +778,21 @@
           }
         }
       }
+    },
+    "xrpl-2025-q1": {
+      "name": "XRPL",
+      "axelarId": "xrpl-2025-q1",
+      "rpc": "wss://s.devnet.rippletest.net:51233",
+      "multisigAddress": "rw6imvkyuZAmCf67xcmCUKuova7vVmB4YB",
```

Comment:
For consistency and ease of automation, could you move this to `contracts.AxelarGateway.address` perhaps? `contracts` is a bit deceiving but makes it convenient to get the right gateway address for any chain

Reply:
Yep, https://github.com/axelarnetwork/axelar-contract-deployments/pull/564/commits/dedd8d8f5f8df582f2507b27f854197d49df4c9f.